code/modules/surgery/organs/internal/ears/_ears.dm

```diff
@@ -159,10 +157,26 @@
 		return FALSE
 	return TRUE
 
-/datum/bodypart_overlay/mutant/cat_ears/color_image(image/overlay, draw_layer, obj/item/bodypart/limb)
-	if(draw_layer != bitflag_to_layer(colorless_layer))
-		return ..()
-	return overlay
+/datum/bodypart_overlay/mutant/cat_ears/get_image(image_layer, obj/item/bodypart/limb)
+	var/mutable_appearance/base_ears = ..()
+
+	// Construct image of inner ears, apply to base ears as an overlay
+	var/gender = (limb?.limb_gender == FEMALE) ? "f" : "m"
+	var/list/icon_state_builder = list()
```

Comment:
Doesn't this also generate null images? earsinner is only a part of EXTERNAL_FRONT layer generation, but seems like it would also attempt to do so for EXTERNAL_BEHIND? (could probably just early return if it's EXTERNAL_BEHIND)

Reply:
from testing I've concluded it doesn't, but given it also doesn't _do anything_ visually we might as well early return, yeah.